Documentation/Administration/Production/ReverseProxy/Index.rst

```diff
@@ -0,0 +1,159 @@
+:navigation-title: Reverse Proxy
+
+..  include:: /Includes.rst.txt
+.. _reverse-proxy-setup:
+
+====================================
+Running TYPO3 behind a reverse proxy
+====================================
+
+Many TYPO3 production environments use a reverse proxy such as NGINX,
+Apache (with mod_proxy), Traefik, or a cloud load balancer. This is especially
+common in container-based deployments using Docker or Kubernetes, where a
+reverse proxy helps manage access to multiple services. Reverse proxies
+typically handle HTTPS termination, request routing, and security features
+such as rate limiting or header filtering.
+
+To ensure TYPO3 works correctly in this setup, some additional configuration is
+required. TYPO3 must be told to trust the proxy headers to interpret the
+correct request scheme and client IP.
```

Comment:
It is not completely clear to me what you want to tell here. I would focus on the things you need to do to run Typo3 behind a loadbalancer/reverse proxy. And step away from all the technical details. If I know I run behind a reverse proxy. What do I need to do to make Typo3 work. 

Maybe mention some things who people can see they run Typo3 behind a reverse proxy. 

I think symfony is doing well on this: https://symfony.com/doc/current/deployment/proxies.html